lib/net/networking_engine.js

```diff
@@ -914,3 +926,17 @@ shaka.net.NetworkingEngine.OnDownloadFailed;
  * @export
  */
 shaka.net.NetworkingEngine.OnRequest;
+
+
+/**
+ * @typedef {function(
+ *    !shaka.net.NetworkingEngine.RequestType,
+ *    (shaka.extern.RequestContext|undefined),
+ *    string,
+ *    string)}
+ *
+ * @description
+ * A callback function called on every request retry
```

Comment:
I know the other descriptions of functions don't say what the parameters are, but at the very least this should specify that the new URI is the first string and the old URI is the second string. We can tell what the other parameters are by context/type, but for two strings it's quite unclear which is which.

Reply:
Added!